### ISSUE TITLE
ERROR in node_modules/ng-otp-input/ng-otp-input.d.ts(5,43): error TS2307: Cannot find module './lib/components/ng-otp-input/ng-otp-input.component'.

### ISSUE THREAD

**Issue by somikdatta** [code-farmz/ng-otp-input#5]
In `ng-otp-input.d.ts`,

`export { NgOtpInputComponent as ɵa } from './lib/components/ng-otp-input/ng-otp-input.component';`

throws error. There is no folder `components` inside `lib` folder

**Comment by code-farmz** [comment]
Hi can you please elaborate the steps to reproduce or may a stackblitz with the problem may help to better understand the problem

**Comment by somikdatta** [comment]
1. installed dependency using `npm install --save ng-otp-input`
2. Imported `NgOtpInputModule` in my `app.module.ts` file and imported it
3. In my HTML template, I used `<ng-otp-input  (onInputChange)="onOtpChange($event)"  [config]="{length:5}"></ng-otp-input>`
4. Executed `ng serve`

`ERROR in node_modules/ng-otp-input/ng-otp-input.d.ts(5,43): error TS2307: Cannot find module './lib/components/ng-otp-input/ng-otp-input.component'.`

5. This error showed up.
6. On manual inspection, I found the `components` folder was not present in the `lib` folder.
7. Fixed the error by manually copying `components` from your repository to my local package.
